core/meta/inc/TInterpreter.h

```diff
@@ -337,6 +337,7 @@ class TInterpreter : public TNamed {
    virtual void   CallFunc_SetFuncProto(CallFunc_t* func, ClassInfo_t* info, const char* method, const std::vector<TypeInfo_t*> &proto, Long_t* Offset, ROOT::EFunctionMatchMode mode = ROOT::kConversionMatch) const = 0;
    virtual void   CallFunc_SetFuncProto(CallFunc_t* func, ClassInfo_t* info, const char* method, const std::vector<TypeInfo_t*> &proto, bool objectIsConst, Long_t* Offset, ROOT::EFunctionMatchMode mode = ROOT::kConversionMatch) const = 0;
 
+   virtual std::string CallFunc_PrintWrapper(CallFunc_t *func) const = 0;
```

Comment:
"Printing" sounds like the wrong verb here. Can we call it `GetWrapper()` or `GetWrapperCode()`